baseplate/clients/memcache/lib.py

```diff
@@ -111,11 +121,13 @@ def dump_and_compress(key, value):  # pylint: disable=unused-argument
             serialized = json.dumps(value)
             flags = Flags.JSON
 
+        serialized_bytes = serialized.encode("utf8")
```

Comment:
bug caught by typing!

`zlib.compress` on py3 takes a bytes object, not a str. we need to encode first. this shows up a few other times in this file too.